stubs/html5lib/html5lib/_inputstream.pyi

```diff
@@ -92,7 +92,7 @@ class HTMLBinaryInputStream(HTMLUnicodeInputStream):
     def reset(self) -> None: ...
     def openStream(self, source): ...
     def determineEncoding(self, chardet: bool = True): ...
-    def changeEncoding(self, newEncoding) -> None: ...
+    def changeEncoding(self, newEncoding: str) -> None: ...
```

Comment:
I think `bytes` or `None` are also accepted here, since the argument is passed to `lookupEncoding()`:

```suggestion
    def changeEncoding(self, newEncoding: str | bytes | None) -> None: ...
```

Reply:
Correct. Things like this make me wish we could reference other methods structures and param types like in TypeScript ^^"